.....
----------------------------------------------------------------------
Ran 5 tests in 0.003s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.003s

OK
FFFFE
======================================================================
ERROR: test_process_csv_data_5 (ClassEval_26_0.CSVProcessorTestProcessCSVData.test_process_csv_data_5)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 279, in test_process_csv_data_5
    self.assertEqual(1, csvProcessor.process_csv_data(4, self.file))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 68, in process_csv_data
    new_title = [title[N]]
                 ~~~~~^^^
IndexError: list index out of range

======================================================================
FAIL: test_process_csv_data_1 (ClassEval_26_0.CSVProcessorTestProcessCSVData.test_process_csv_data_1)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 242, in test_process_csv_data_1
    self.assertEqual(read_title, title)
AssertionError: Lists differ: ['a'] != ['a', 'b', 'c', 'd']

Second list contains 3 additional elements.
First extra element 1:
'b'

- ['a']
+ ['a', 'b', 'c', 'd']

======================================================================
FAIL: test_process_csv_data_2 (ClassEval_26_0.CSVProcessorTestProcessCSVData.test_process_csv_data_2)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 252, in test_process_csv_data_2
    self.assertEqual(read_title, title)
AssertionError: Lists differ: ['b'] != ['a', 'b', 'c', 'd']

First differing element 0:
'b'
'a'

Second list contains 3 additional elements.
First extra element 1:
'b'

- ['b']
+ ['a', 'b', 'c', 'd']

======================================================================
FAIL: test_process_csv_data_3 (ClassEval_26_0.CSVProcessorTestProcessCSVData.test_process_csv_data_3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 262, in test_process_csv_data_3
    self.assertEqual(read_title, title)
AssertionError: Lists differ: ['c'] != ['a', 'b', 'c', 'd']

First differing element 0:
'c'
'a'

Second list contains 3 additional elements.
First extra element 1:
'b'

- ['c']
+ ['a', 'b', 'c', 'd']

======================================================================
FAIL: test_process_csv_data_4 (ClassEval_26_0.CSVProcessorTestProcessCSVData.test_process_csv_data_4)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 272, in test_process_csv_data_4
    self.assertEqual(read_title, title)
AssertionError: Lists differ: ['d'] != ['a', 'b', 'c', 'd']

First differing element 0:
'd'
'a'

Second list contains 3 additional elements.
First extra element 1:
'b'

- ['d']
+ ['a', 'b', 'c', 'd']

----------------------------------------------------------------------
Ran 5 tests in 0.006s

FAILED (failures=4, errors=1)
F
======================================================================
FAIL: test_main (ClassEval_26_0.CSVProcessorTestMain.test_main)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_26_0.py", line 309, in test_main
    self.assertEqual(read_title, title)
AssertionError: Lists differ: ['a'] != ['a', 'b', 'c', 'd']

Second list contains 3 additional elements.
First extra element 1:
'b'

- ['a']
+ ['a', 'b', 'c', 'd']

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)
.....
----------------------------------------------------------------------
Ran 5 tests in 0.071s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.017s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.006s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.007s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.008s

OK
.
----------------------------------------------------------------------
Ran 1 test in 0.002s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.002s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.005s

OK
..F..
======================================================================
FAIL: test_process_json_3 (ClassEval_50_0.JSONProcessorTestProcessJsonExistingKey.test_process_json_3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_50_0.py", line 255, in test_process_json_3
    self.assertEqual(self.processor.process_json(self.file_path, remove_key), 0)
AssertionError: -1 != 0

----------------------------------------------------------------------
Ran 5 tests in 0.005s

FAILED (failures=1)
.
----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.007s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.005s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.003s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.003s

OK
.
----------------------------------------------------------------------
Ran 1 test in 0.002s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.004s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.007s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.006s

OK
.....
----------------------------------------------------------------------
Ran 5 tests in 0.004s

OK
.
----------------------------------------------------------------------
Ran 1 test in 0.002s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.012s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.011s

OK
......
----------------------------------------------------------------------
Ran 6 tests in 0.012s

OK
FFFFF.
======================================================================
FAIL: test_create_zip_file_1 (ClassEval_99_0.ZipFileProcessorTestCreateZipFile.test_create_zip_file_1)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 492, in test_create_zip_file_1
    self.assertTrue(success)
AssertionError: False is not true

======================================================================
FAIL: test_create_zip_file_2 (ClassEval_99_0.ZipFileProcessorTestCreateZipFile.test_create_zip_file_2)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 515, in test_create_zip_file_2
    self.assertTrue(success)
AssertionError: False is not true

======================================================================
FAIL: test_create_zip_file_3 (ClassEval_99_0.ZipFileProcessorTestCreateZipFile.test_create_zip_file_3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 538, in test_create_zip_file_3
    self.assertTrue(success)
AssertionError: False is not true

======================================================================
FAIL: test_create_zip_file_4 (ClassEval_99_0.ZipFileProcessorTestCreateZipFile.test_create_zip_file_4)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 561, in test_create_zip_file_4
    self.assertTrue(success)
AssertionError: False is not true

======================================================================
FAIL: test_create_zip_file_5 (ClassEval_99_0.ZipFileProcessorTestCreateZipFile.test_create_zip_file_5)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 583, in test_create_zip_file_5
    self.assertTrue(success)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 6 tests in 0.007s

FAILED (failures=5)
F
======================================================================
FAIL: test_ZipFileProcessor (ClassEval_99_0.ZipFileProcessorTest.test_ZipFileProcessor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/snigdhaghoshdastidar/Desktop/CS540/classeval_evaluation/ClassEval_99_0.py", line 623, in test_ZipFileProcessor
    self.assertTrue(success)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 1 test in 0.002s

FAILED (failures=1)
